bestFitLine/bestFitLine.js

```diff
@@ -52,12 +52,12 @@ AmCharts.bestFitLineProcess = function( chart, validateData ) {
 				var cat = getCategoryIndex( chart.dataProvider[ z ][ chart.categoryField ], z, chart );
 
 				// get value
-				var value = chart.dataProvider[ z ][ graph.valueField ];
-				if ( value !== undefined ) {
+				var value = parseFloat( chart.dataProvider[ z ][ graph.valueField ] );
```

Comment:
Rather than `parseFloat`, it may be better to use `+` instead, like this:

```
var value = +chart.dataProvider[ z ][ graph.valueField ];
```

The `parseFloat` function has all sorts of weird gotchas. The `+` unary operator is more reliable.

However, `+null` returns `0`, which isn't good. So you should probably add in a `null` check, like this:

```
var value = chart.dataProvider[ z ][ graph.valueField ];

if ( value != null ) {
  value = +value;

  // rest of code goes here...
}
```